site/layouts/docs/single.html

```diff
@@ -8,8 +8,5 @@ <h1>{{.Title}}</h1>
         {{ end }}
     </article>
 
-    <p>
-        <a href="{{ "/" | relURL }}" rel="home">
-        &larr; Table of Contents</a>
-    </p>
+    <a href="{{  "/docs/" | relURL}}" class="btn btn-link btn-sm rounded">&larr; Documentation Home</a>
```

Comment:
Navigation is a bit weird. The flow is

Landing page 
 --click--> User docs
 --click--> User docs overview
 --click--> User docs page
 --click `Documentation Home`--> User+Developer docs overview (aka Documentation)

I would have expected that the last step brings me back to "User docs overview"

Reply:
Addressing in top-level comment.